draft-packages/collapsible/KaizenDraft/ExpertAdviceCollapsible/ExpertAdviceCollapsible.module.scss

```diff
@@ -0,0 +1,50 @@
+@import "~@kaizen/design-tokens/sass/border";
+@import "~@kaizen/design-tokens/sass/color";
+@import "~@kaizen/design-tokens/sass/spacing";
+
+$heading-active-color: $color-purple-100;
+
+.expertAdviceContainer {
+  background-color: $color-purple-100;
+  border: 1px $border-solid-border-style $color-purple-300;
```

Comment:
I just realised the design has the border at 2px, so we'll just need to update this :)

Reply:
Now updated to 2px.